src/app/pages/TopicPage/TopicPage.jsx

```diff
@@ -86,26 +88,35 @@ const TopicPage = ({ pageData }) => {
             {description && <TopicDescription>{description}</TopicDescription>}
           </div>
           {curations.map(
-            ({
-              visualProminence,
-              summaries,
-              curationId,
-              title: curationTitle,
-              link,
-              position,
-            }) => (
-              <Curation
-                headingLevel={curationTitle && 3}
-                key={curationId}
-                visualStyle={VISUAL_STYLE.NONE}
-                visualProminance={visualProminence}
-                promos={summaries}
-                title={curationTitle}
-                topStoriesTitle={topStoriesTitle}
-                position={position}
-                link={link}
-                curationLength={curations && curations.length}
-              />
+            (
+              {
+                visualProminence,
+                summaries,
+                curationId,
+                title: curationTitle,
+                link,
+                position,
+                visualStyle,
+              },
+              index,
+            ) => (
+              // eslint-disable-next-line react/no-array-index-key
+              <React.Fragment key={`${curationId} ${index}`}>
+                {visualStyle === VISUAL_STYLE.BANNER && !isLive() && (
+                  <MessageBanner title={curationTitle} summaries={summaries} />
+                )}
+                <Curation
```

Comment:
This should be an if/else ternary
```suggestion
                {shouldRenderMessageBanner ? (
                  <MessageBanner title={curationTitle} summaries={summaries} />
                ) : (
                <Curation
                // ... rest of code here
                }
```

Reply:
Done ✅